src/components/CameraPermissions/Primer/style.css

```diff
@@ -50,11 +50,10 @@
   background-image: url('assets/allow.svg');
   background-repeat: no-repeat;
   background-size: 100%;
-  position: relative;
-  flex: 0 1 100%;
-  /* 90.2% is the aspect ratio of responsive div */
-  /* The percentage has been calculated by dividing the original height value divided by the original width value. In this case 94.67/105.01 = 0.992 */
-  padding-top: 90.2%;
+  background-position: center;
+  height: 136*@unit;
+  width: 136*@unit;
+  min-width: 16%;
```

Comment:
Why are you using a percentage for the width?

Reply:
Was trying to set a minimum width based on viewport, but now I realise it's not required since width is already defined.